openrouter.ts

```
@@ -0,0 +1,90 @@
+              // Ignore invalid JSON
+            console.warn("failed to parse SSE data",data,e);
+            
+            }
+          }
+        }
+      }
+    } finally {
+      resolve()
+      reader.cancel();
+    }
+  })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)